tests/test_code_gen.py

```diff
@@ -19,85 +18,100 @@
 
 import astor
 
+try:
+    from astunparse_common import AstunparseCommonTestCase
+except ImportError:
+    AstunparseCommonTestCase = None
 
 def canonical(srctxt):
     return textwrap.dedent(srctxt).strip()
 
 
-class CodegenTestCase(unittest.TestCase):
+class Comparisons(object):
+
+    to_source = staticmethod(astor.to_source)
 
-    def assertAstEqual(self, srctxt):
+    assertSrcEqual = unittest.TestCase.assertEqual
+
+    def assertAstEqual(self, ast1, ast2):
+        dmp1 = astor.dump_tree(ast1)
+        dmp2 = astor.dump_tree(ast2)
+        self.assertEqual(dmp1, dmp2)
+
+    def roundtrip_ast(self, srctxt):
```

Comment:
Sounds good.  I renamed them because the name didn't describe the functionality, and I wanted to insert underlying functionality that the original names better matched, but you're right -- it's still an assertion